src/uct/ib/rc/accel/rc_mlx5_ep.c

```diff
@@ -548,6 +578,10 @@ ucs_status_t uct_rc_mlx5_ep_flush(uct_ep_h tl_ep, unsigned flags,
         return UCS_OK;
     }
 
+    if (ucs_unlikely(flags & UCT_FLUSH_FLAG_KEEPALIVE)) {
```

Comment:
probably it will be used more frequently than normal flush, i'd remove `unlikely`